content/pt-br/docs/reference/access-authn-authz/authorization.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Autorização
+content_type: concept
+weight: 60
+---
+
+<!-- overview -->
+Aprenda mais sobre autorização no Kubernetes, incluindo detalhes sobre
+criação de politicas utilizando módulos de autorização suportados.
+
+<!-- body -->
+
+No Kubernetes, você deve estar autenticado (conectado) antes que sua requisição possa ser
+autorizada (permissão concedida para acesso). Para obter informações sobre autenticação,
+visite [Controlando Acesso à API do Kubernetes](/pt-br/docs/concepts/security/controlling-access/).
+
+O Kubernetes espera atributos que são comuns a requisições de APIs REST. Isto significa
+que autorização no Kubernetes funciona com sistemas de controle de acesso a nível de organizações
+ou de provedores de nuvem que possam lidar com outras APIs além das APIs do Kubernetes.
+
+## Determinar se uma requisição é permitida ou negada
+
+Kubernetes autoriza requisições de API utilizando o servidor de API. Ele avalia
+todos os atributos de uma requisição em relação a todas as políticas disponíveis a permite ou nega a requisição.
+Todas as partes de uma requisição de API deve ser permitido por alguma política para que possa prosseguir.
+Isto significa que permissões sao negadas por padrão.
+
+(Embora o Kubernetes use o servidor de API, controles de acesso e políticas que
+dependem de campos específicos de tipos específicos de objetos são tratados pelo Admission
+Controller.)
+
+Quando múltiplos modules de autorização são configurados, cada um será verificado em sequência.
+Se qualquer dos autorizadores aprovarem ou negarem uma requisição, a decisão é imediatamente
+retornada e nenhum outro autorizador é consultado. Se todos os módulos de autorização não tiverem
+nenhuma opinião sobre requisição, então a requisição é negada. Uma negação retorna um
```

Comment:
How do you feel about rephrasing "Se todos os módulos de autorização não tiverem..." negatively, i.e. "se nenhum módulo de autorização tiver..."?
It would sound more idiomatic.

Reply:
Perfect, thanks~!